api.ts

```diff
@@ -137,7 +137,7 @@ export interface SourceFileConfiguration {
     /**

```

Comment:
We also need to update the version number as part of this change.  I can help you with this.

Reply:
Ok, yes thank you